djangocms_admin_style/sass/components/_icons.scss

```diff
@@ -177,6 +177,9 @@ a[id*="clocklink"] {
 .delete-related {
     @include button-variant($btn-default-color, $btn-default-bgcolor, $btn-default-border, true);
     display: inline-block;
+    /* make sure that add another button has correct width and height on advanced setting */
```

Comment:
``// make sure that add another button has correct width and height on advanced setting #251``